### PR TITLE
atop: upgrade version 2.6.0 -> 2.9.0

### PR DESCRIPTION
**Pull request by mfrw**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [ ] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
Upgrade version 2.6.0 -> 2.9.0

###### Change Log  

- Upgrade version 2.6.0 -> 2.9.0

###### Does this affect the toolchain?  

**NO**

###### Associated issues  

- NA

###### Links to CVEs  
- NA

###### Test Methodology

- Pipeline build id: [PR-7178](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=478312&view=results)
Local Build:
image

### CODE REVIEW

File: SPECS/atop/format.patch [hunk 1]
```diff
@@ -0,0 +1,33 @@
+--- drawbar.c~	2023-05-13 03:20:27.000000000 -0500
```
Comment (reubeno) [inline SPECS/atop/format.patch:1]: Did we spin this patch ourselves? It looks like this was fixed [upstream](https://github.com/Atoptool/atop/commit/957ff648436fa4a6f08ad9a8c5ea856a5f33ef5b) perhaps? 

Incidentally, it also looks like v2.10.0 came out this week and includes the above linked change.

Comment (mfrw) [reply SPECS/atop/format.patch:1]: Yes a new version is up but it requires glib2 which we do not ship.
Should we add glib2 as well to move to the latest version ?

Comment (reubeno) [reply SPECS/atop/format.patch:1]: The `glib.spec` in CBL-Mariner provides `glib2`. Is there a specific version of glib2 that's required?

File: SPECS/atop/atop.spec [hunk 2]
```diff
@@ -93,6 +91,10 @@ install -Dp -m 0644 atop-rotate.* %{buildroot}%{_unitdir}/
 %{_sbindir}/atopacctd
 
 %changelog
+* Fri Jan 05 2024 Muhammad Falak <mwani@microsoft.com> -- 2.9.0-1
```
Comment (anphel31) [inline SPECS/atop/atop.spec:94]: remove extra -

**Review by anphel31 — APPROVED** [review]
After fixing some buddybuild issues; reran a new buddybuild: https://dev.azure.com/mariner-org/mariner/_build/results?buildId=486958&view=results

File: SPECS/atop/format.patch (hunk 1 above)
Comment (mfrw) [reply SPECS/atop/format.patch:1]: Yes, thank you. It seem that upgrade to 2.10.0 is non-trivial, for now I will merge this and circle back when I get some additional cycles.

**Review by mfrw — COMMENTED** [review]
New test build:
- [PR-7178](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=487041&view=results)

**Pull request merged**